### PR TITLE
Fix multi case study plots and tables

### PR DESCRIPTION
**Pull request by Sinerum**
Plots and tables containing multiple case studies no longer crash when generating the filename.
Their file name is now based on all projects in the provided case studies

### CODE REVIEW

**Review by boehmseb — APPROVED** [review]

**Review by vulder — CHANGES_REQUESTED** [review]
Fix looks good but could you please remove the pylint errors before merging :)

File: varats/varats/table/table.py [hunk 1]
```diff
@@ -154,7 +154,7 @@ def table_file_name(self, table_format: TableFormat) -> str:
                 table_ident = f"{case_study.project_name}_{case_study.version}_"
             else:
                 table_ident = \
-                f"{reduce(lambda x, y: f'{x}{y.project_name}_', case_study, '')}"
+            f"{reduce(lambda x, y: f'{x}{y.project_name}_', case_study, '')}"
```
Comment (vulder) [inline varats/varats/table/table.py]: Sorry to be a bit picky here, but we should the string or somehow else make the line fit. The problem with this formatting ist, da it looks like the `f` string is on the level of the outer if case.

Comment (Sinerum) [reply varats/varats/table/table.py]: I don't like this either, the only options i see is to shorten the name of the local variable case_study to something like cs_list or extract this String into its own method.

Comment (vulder) [reply varats/varats/table/table.py]: You can also just move the reduce out of the f-string and create a local variable. That seems like the simplest solution

Comment (Sinerum) [reply varats/varats/table/table.py]: Or even better just remove the f string as the reduce returns a string 🤦

**Review by vulder — APPROVED** [review]

**Pull request merged**